### ISSUE TITLE
Process used once - claimed used twice

### ISSUE THREAD

**Issue by avecchio** [nextflow-io/nextflow#4084]
[main.txt](https://github.com/nextflow-io/nextflow/files/11994420/main.txt)
### Expected behavior and actual behavior

The expected behavior is for the nextflow pipeline to execute as all processes defined in the workflow have been defined and used only once. 

The actual behavior is that Nextflow throws an error for only the process 'skesa_consensus_contig_map' claiming the process has been used twice in the workflow. In reviewing the file numerous times, the process has only been defined and used only once. This is rather peculiar as Nextflow does not thrown an error for a similarly named process 'spades_consensus_contig_map' which should be nearly identical to 'skesa_consensus_contig_map'. 

### Steps to reproduce the problem

The Nextflow pipeline code is attached. To replicate the issue, run 'nextflow run main.nf' on the pipeline. No additional configurations/arguments should be required. Note* there are some local directories referenced and may need to be changed.

```
N E X T F L O W  ~  version 23.04.2
Launching `pipeline.nf` [serene_shaw] DSL2 - revision: 6d53ae38d5
[-        ] process > decompress_reads           -
[-        ] process > qc_reads                   -
[-        ] process > extract_fastqc_report      -
[-        ] process > trim_reads                 -
[-        ] process > consensus_genome_reads_map -
Process 'skesa_consensus_contig_map' has been already used -- If you need to reuse the same component, include it with a different name or include it in a different workflow context

 -- Check script 'pipeline.nf' at line: 582 or see '.nextflow.log' file for more details
Cannot find any reads matching: /home/ubuntu/contig/data/*_{1,2}.fastq.gz
```

### Environment 

* Nextflow version: 23.04.2 build 5870
* Java version: Groovy 3.0.16 on OpenJDK 64-Bit Server VM 11.0.13+7-b1751.21
* Operating system: Linux
* Bash version: 5.0.17(1)-release (x86_64-pc-linux-gnu)

**Comment by bentsherman** [comment]
Missing comma strikes again:

```groovy
process skesa_consensus_contig_map {
    // ...

    input:
    tuple val(sraId), path(trimmedPair1), path(trimmedPair2) path(skesaAssembly)

    // ...
}
```

**Comment by bentsherman** [comment]
Duplicate of #3185 

However, I think I may have finally found a fix for this particular error...
